src/Block/RssBody.jsx

```diff
@@ -10,8 +10,8 @@ const RssBody = ({ data, isEditMode }) => {
     let parser = new Parser();
     setFeedItems([]);
     if (data?.feed?.length > 0) {
-      let base_url = config.settings.apiPath;
-      parser.parseURL(base_url + '/@get_rss_feed?feed=' + data.feed, function (
+      let base_url = config.settings.devProxyToApiPath;
```

Comment:
Now that I read this with more attention to details, I think ``devProxyToApiPath`` could be wrong here. It should only be used in development, it will not be there in production.

Reply:
I agree with @pnicolli